core/domain/base_model_validators.py

```diff
@@ -307,14 +307,16 @@ def _validate_model_time_fields(cls, item):
         Args:
             item: ndb.Model. Entity to validate.
         """
-        if item.created_on > item.last_updated:
+        if item.created_on > (
```

Comment:
Add a comment above this: "We add a tolerance of 1 second because, for some models, created_on and last_updated are the same time, and occur within milliseconds of each other. In such cases, created_on might end up being very slightly greater than last_updated."

Also, on second thoughts, let's change the tolerance to 1 second. I think that should be sufficient.

Reply:
Done.